src/main/java/ieit/agh/edu/pl/botcompetitionarena/domain/bot/boundary/BotController.java

```diff
@@ -59,7 +82,7 @@ public BotController(BotService botService) {
     @Transactional
     @PostMapping("/set-content/{id}")
     public ResponseEntity<Object> setBot(@PathVariable("id") Long botId,
-                                            @RequestParam("payload") MultipartFile payload) {
+                                         @RequestParam("payload") MultipartFile payload) {
```

Comment:
"payload" i właściwie wszystkie inne stałe, które mamy w parametrach i ścieżkach query można w sumie powyciągać do stałych:
`private final String PAYLOAD_PARAM = "payload";`
`private final String ID_PATH = "id";` 
(pozostałe analogicznie)
tak, żeby m.in. uniknąć literówek lub w razie potrzeby zmienić tylko w jednym miejscu